docs/getting-started/tutorials/table_api.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Table API"
+nav-id: tableapitutorials
+nav-title: 'Table API'
+nav-parent_id: apitutorials
```

Comment:
Does it make sense to already put this under it's one section "Code Walkthroughs"?